services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -45,65 +45,72 @@ def setup_logger(settings: DynamicSidecarSettings):
     logging.root.setLevel(settings.log_level)
 
 
-def assemble_application() -> FastAPI:
-    """
-    Creates the application from using the env vars as a context
-    Also stores inside the state all instances of classes
-    needed in other requests and used to share data.
-    """
+def create_basic_app() -> FastAPI:
```

Comment:
Minor: maybe minimal (as you put it below in a comment) could sound better. When I read it, `minimal app` gives me a better understanding than basic (which does not make me thing of "small" nor "the basis for").
Another alternative is `base app` which gives the impression of being the core for something else.

Reply:
OK. will do in next PR